packages/blade/src/utils/usePaymentCardDetection.tsx

```diff
@@ -0,0 +1,102 @@
+import React from 'react';
+import { CreditCardIcon } from '~components/Icons';
+
+export type PaymentCardBrand = 'visa' | 'mastercard' | 'amex' | 'unknown';
+
+/**
+ * Detects the payment card brand based on the card number
+ * @param cardNumber - The card number to analyze
+ * @returns The detected payment card brand
+ */
+export const detectPaymentCardBrand = (cardNumber: string): PaymentCardBrand => {
```

Comment:
Are we exposing this utility to consumers? 

If so let's review this utility ka API ekbar for 5min tommorow standup.